installation/docker.adoc

```diff
@@ -10,4 +10,6 @@
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+
+Starting in a later beta release, you will be able to install OpenShift Enterprise 3.0 Beta using Docker images. Images will be made available for OpenShift platform components such as masters, etcd, nodes, and routers.
```

Comment:
For @CowboysFan as well: We have to be careful about how and when we use specific product names in the docs. Remember that the same doc set will be used for three products, so unless you are explicitly working within one of the version-specific ifdefs, I would suggest sticking with just "OpenShift".